manifests/f/Fossil/Fossil/2.20/Fossil.Fossil.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Fossil.Fossil
+PackageVersion: "2.20"
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\fossil.exe
+    PortableCommandAlias: fossil
+  InstallerUrl: https://fossil-scm.org/home/uv/fossil-w64-202302092100.zip
+  InstallerSha256: 4C9B56CD1AA3E3AC13478325B15E2EB36DBCEB3E13BB4F8D94BA055D73139F27
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\fossil.exe
+    PortableCommandAlias: fossil
+  InstallerUrl: https://fossil-scm.org/home/uv/fossil-w32-202302092100.zip
+  InstallerSha256: C96195CA402F51FA6650C3241E952699A3AAEB26491114D21DDFC883299DDEF4
```

This is somehow matching the x64's hash.
```suggestion
  InstallerSha256: E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855
```